packages/docs/src/examples/v-command-palette/contained.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <v-container class="d-flex flex-column align-center justify-center">
+    <v-card class="align-self-stretch mb-3">
+      <v-card-title>Contained Palette</v-card-title>
+      <v-card-text>
+        The VCommandPalette can be rendered inline within any component using the <code>contained</code> prop.
+      </v-card-text>
+    </v-card>
+    <v-command-palette
+      :close-on-execute="false"
+      :items="items"
+      :model-value="true"
+      class="align-self-stretch mx-auto"
+      hotkeys-scope="focused"
```

Comment:
I have opened the docs page and `contained` examples still hijack keyboard up/down/backspace.